spec/abstractops.html

```diff
@@ -1436,12 +1439,16 @@ <h1>
           1. Let _offset_ be the source text matched by the |UTCOffset| Parse Node contained within _parseResult_.
           1. Set _timeZoneResult_.[[OffsetString]] to CodePointsToString(_offset_).
       1. Let _calendar_ be *undefined*.
+      1. Let _calendarWasCritical_ be *false*.
       1. For each |Annotation| Parse Node _annotation_ contained within _parseResult_, do
         1. Let _key_ be the source text matched by the |AnnotationKey| Parse Node contained within _annotation_.
         1. If CodePointsToString(_key_) is *"u-ca"*, then
           1. If _calendar_ is *undefined*, then
             1. Let _value_ be the source text matched by the |AnnotationValue| Parse Node contained within _annotation_.
             1. Let _calendar_ be CodePointsToString(_value_).
+            1. If _annotation_ contains an |AnnotationCriticalFlag| Parse Node, set _calendarWasCritical_ to *true*.
+          1. Else,
+            1. If _annotation_ contains an |AnnotationCriticalFlag| Parse Node, or _calendarWasCritical_ is *true*, throw a *RangeError* exception.
```

Comment:
Doesn't the polyfill throw a SyntaxError here?

Reply:
(see other comment; that's not part of the polyfill, it's just a quirk of the way the grammar is tested)